trade-module/src/main/java/io/edpn/backend/trade/application/dto/web/filter/LocateCommodityFilterDto.java

```diff
@@ -1,24 +1,24 @@
 package io.edpn.backend.trade.application.dto.web.filter;
 
-public interface LocateCommodityFilterDto {
-    
+public interface LocateCommodityFilterDto extends PageFilterDto {
+
     String commodityDisplayName();
-    
+
     Double xCoordinate();
-    
+
     Double yCoordinate();
-    
+
     Double zCoordinate();
```

Comment:
BTW do X, Y and Z coordinates make much sense independently? If not, shouldn't we have a `Coordinates` object instead?